aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,134 @@
+
+5. Complete the form with the following values:
+    
+    | Setting                       | Value                                             |
+    |-------------------------------|---------------------------------------------------|
+    | **Domain Name**               | *\<the domain name of your B2C tenant\>*          |
+    | **Application ID**            | *\<paste the Application ID from the clipboard\>* |
+    | **Callback Path**             | *\<use the default value\>*                       |
+    | **Sign-up or sign-in policy** | `B2C_1_SiUpIn`                                    |
+    | **Reset password policy**     | `B2C_1_SSPR`                                      |
+    | **Edit profile policy**       | *\<leave blank\>*                                 |
+
+    Select the **Copy** link next to **Reply URI** to copy the Reply URI to the clipboard. Select **OK** to close the **Change Authentication** dialog, and then select **OK** to create the web app.
+
+## Finish the B2C app registration
+
+Return to the browser window with the B2C app properties still open. Change the temporary **Reply URL** specified earlier to the value copied from Visual Studio, and then select **Save** at the top of the window.
+
+> [!TIP]
+> If you didn't copy the Reply URL, use the SSL address from the Debug tab in the web project properties, and append the **CallbackPath** value from *appsettings.json*.
+
+## Configure policies
+
+Use the steps in the Azure AD B2C documentation to [create a sign-up or sign-in policy](/azure/active-directory-b2c/active-directory-b2c-reference-policies#create-a-sign-up-or-sign-in-policy), and then [create a password reset policy](/azure/active-directory-b2c/active-directory-b2c-reference-policies#create-a-password-reset-policy). Use the example values provided in the documentation for **Identity providers**, **Sign-up attributes**, and **Application claims**. Using the **Run now** button to test the policies as described in the documentation is optional.
+
+> [!WARNING]
+> Ensure the policy names are exactly as described in the documentation, as those policies were used in the **Change Authentication** dialog in Visual Studio. The policy names can be verified in *appsettings.json*.
+
+## Run the app
+
+In Visual Studio, press **F5** to build and run the app. After the web app launches, select **Sign in**.
+
+![Sign into the app](./azure-ad-b2c/_static/signin.png)
+
+The browser redirects to the Azure AD B2C tenant. Sign in with an existing account (if one was created testing the policies) or select **Sign up now** to create a new account. The **Forgot your password?** link is used to reset a forgotten password.
+
+![Azure AD B2C login](./azure-ad-b2c/_static/b2csts.png)
+
+After successfully signing in, the browser redirects to the web app.
+
+![Success](./azure-ad-b2c/_static/success.png)
+
+## Next steps
+
+In this tutorial, you will learned how to:
```

Comment:
Here, too. No "you" ... I'd hate to see you fall prey to a *Documentation Death Squad*! :boy: :gun: :smile: ... and of course the "will learned" needs a touch. Using past tense here is probably a reasonable exception to their 'use present tense' guidance.

> This tutorial covered:
>
> \* Creation of an Azure AD B2C tenant.
> \* Registration of an app in Azure AD B2C.
> \* Use of Visual Studio to create an ASP.NET Core web app configured to use the Azure AD B2C tenant for authentication.
> \* Configuration of policies controlling the behavior of the Azure AD B2C tenant.

Reply:
It's funny you mention this!  @Rick-Anderson had already made me well-aware of this rule, and it was my mantra the entire time I wrote this article.  Then @JRAlexander made a comment about using the MVC (writing guidelines, not web framework) templates, so I went back and copied/pasted the boilerplate text...  Including both of these erroneous uses of "you."  😃